src/server.cpp

```diff
@@ -1523,16 +1536,13 @@ void Server::SendBreath(u16 peer_id, u16 breath)
 	Send(&pkt);
 }
 
-void Server::SendAccessDenied(u16 peer_id, AccessDeniedCode reason, const std::string &custom_reason)
+void Server::SendAccessDenied(u16 peer_id, AccessDeniedCode reason,
+		const std::string &custom_reason, bool reconnect)
 {
 	DSTACK(__FUNCTION_NAME);
 
 	NetworkPacket pkt(TOCLIENT_ACCESS_DENIED, 1, peer_id);
-	pkt << (u8) reason;
-
-	if (reason == SERVER_ACCESSDENIED_CUSTOM_STRING) {
-		pkt << custom_reason;
-	}
+	pkt << (u8) reason << custom_reason << (u8)reconnect;
```

Comment:
Bypassing the goal of SERVER_ACCESSDENIED_CUSTOM_STRING by allowing other codes to send custom strings remove the interest of CUSTOM STRING and the interest of static others. I don't see why somebody want to change the message: "Bad password" by another. We must have the same standard error messages on all server on the network negocitiation. prejoin and join callback from Lua permit to change the purpose if somes want to change things.